### ISSUE TITLE
exa doesn't sort explicit arguments?

### ISSUE THREAD

**Issue by melissaboiko** [ogham/exa#1053]
Putting this as a question because I don't know if it's intended, or if I'm missing something:

    > touch older newer
    > ls -t older newer
    newer  older
    > ls -tr older newer
    older  newer
    > exa -sold older newer
    older  newer
    > exa -snew older newer
    older  newer
    > exa -snew newer older
    newer  older

sorting the arguments is useful for scripting.

**Comment by ariasuni** [comment]
Yeah I think it’s not ideal, and if I remember correctly, that’s one of the thing I was trying to fix in #869.
